package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "space-travelers-hub",
+  "version": "1.0.0",
+  "description": ">In this task, we worked with the real live data from the SpaceX API. The task was to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] I was unable to test your application to see if everything is working as expected. However, I noticed your package.json is missing the correct scripts needed for running your app or even the tests. Please see screenshots of the error message I got and I will suggest you try to amend them.

`npm start` command error screenshot|
:-----------------------------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/163586124-321a2fb4-3a4b-4dcf-9c16-d45e408e9ad5.png)

`npm test` command error screenshot|
:------------------------------------------------:
![a2](https://user-images.githubusercontent.com/39523312/163586135-708852c7-967b-4eba-8730-34467673dab1.png)
|

Reply:
Hello @mmsesay , Sorry for the issue, I have corrected it and here below is the look on the new  output.
<img width="1315" alt="Screen Shot 2022-04-15 at 17 47 31" src="https://user-images.githubusercontent.com/82823425/163592297-2804f1d1-ba33-4bab-ad16-b84804f85f4f.png">

Reply:
Good job @Paskab012. Please ask for a re-review when your team is ready. Keep rocking mate. 👍

Reply:
Hi @keza681, @Paskab012

I am still missing the scripts to run or test your application. Thus, I am still experiencing the same issue demonstrated by the previous reviewer. ***Kindly push the changes you made.***

I will give you an account of what procedures I took to try to run your application.
Normally the "scripts" property should look like this on the package.json file.
![Screenshot from 2022-04-15 18-43-50](https://user-images.githubusercontent.com/38088818/163598534-73b28bb4-0279-4e9e-93c9-66f898568c1e.png)

But yours looks like this:
![Screenshot from 2022-04-15 18-52-46](https://user-images.githubusercontent.com/38088818/163598486-931fbad4-3dea-4f34-82d7-ce5fd013596b.png)

I took the liberty of adding the scripts to try and get it to work. Then I received this error: 🥲 
![Screenshot from 2022-04-15 18-43-16](https://user-images.githubusercontent.com/38088818/163598733-74a86c2d-850d-4ea2-a776-959a5b0ecce6.png)
Then I resolved the issue by installing the missing `react-scripts` locally.
```
npm install --save react-scripts
```
After that, I was able to start your application. However, the was still a compile issue: 😿 
![Screenshot from 2022-04-15 18-47-07](https://user-images.githubusercontent.com/38088818/163599251-10abe533-3756-4063-8117-0dd61876a2a0.png)

And, when I ran the tests I received this:
![Screenshot from 2022-04-15 18-47-48](https://user-images.githubusercontent.com/38088818/163599351-43045df0-280f-4820-9f50-e591000ccc0c.png)

Kindly look into this or otherwise inform us how you are running your application and tests without scripts.
 ***Remember to push the changes you made*** 😄

Reply:
Thanks @KDlamini , @mmsesay,  for those reviewers; we actually encountered a blocker on version regarding react while Installing it. So this , resulted to non response of some some react packages helping to start and to test. But now the problem is solved and everything is working perfectly. `When testing the app, Kindly make sure you are on the develop branch as everything is now merged there` . Deep apologies for that☺️🙏🏼
## KINDLY USE `NPM` TO RUN AND TEST THE APP

## Look the Output 
<img width="1402" alt="Screen Shot 2022-04-15 at 20 20 29" src="https://user-images.githubusercontent.com/82823425/163607092-eff98ac4-e493-4667-861d-8379544bafe2.png">